pulser-core/pulser/sequence/_schedule.py

```diff
@@ -257,3 +350,12 @@ def add_target(self, qubits_set: set[QubitId], channel: str) -> None:
         self[channel].slots.append(
             _TimeSlot("target", ti, tf, set(qubits_set))
         )
+
+    def wait_for_fall(self, channel: str) -> None:
+        """Adds a delay to let the channel's amplitude ramp down."""
+        fall_time = (
+            self[channel].get_duration(include_fall_time=True)
+            - self[channel].get_duration()
+        )
+        if fall_time > 0:
+            self.add_delay(self[channel].adjust_duration(fall_time), channel)
```

Comment:
I assume this is to wait for a pulse that takes longer to go to zero because of modulation. Could you add a comment about it?